aiida/backends/sqlalchemy/migrations/versions/1feaea71bd5a_migrate_repository.py

```diff
@@ -111,8 +111,8 @@ def upgrade():
     container_id = backend.get_repository().uuid
     statement = text(
         f"""
-        INSERT INTO db_dbsetting (key, val, description)
-        VALUES ('repository|uuid', to_json('{container_id}'::text), 'Repository UUID')
+        INSERT INTO db_dbsetting (key, val, description, time)
+        VALUES ('repository|uuid', to_json('{container_id}'::text), 'Repository UUID', NOW())
```

Comment:
Will this leave existing databases, who have already applied this migration, in a broken state? They will not have this column set. I think this might require a separate migration that sets an arbitrary value for the `time` column on existing rows.

Reply:
True, so this change is fine.